Client/sdk/game/CPools.h

```diff
@@ -107,12 +108,7 @@ class CPools
     virtual void ResetPedPoolCount() = 0;
     virtual void InvalidateLocalPlayerClientEntity() = 0;
 
-    virtual uint AllocateTextureDictonarySlot(uint uiSlotID, std::string& strTxdName) = 0;
-    virtual void RemoveTextureDictonarySlot(uint uiTxdID) = 0;
-    virtual bool IsFreeTextureDictonarySlot(uint uiTxdID) = 0;
-
-    virtual ushort GetFreeTextureDictonarySlot() = 0;
-
     virtual CBuildingsPool& GetBuildingsPool() noexcept = 0;
-    virtual CDummyPool&   GetDummyPool() noexcept = 0;
+    virtual CDummyPool&     GetDummyPool() noexcept = 0;
+    virtual CTxdPool&       GetTxdPool() noexcept = 0;
```

Comment:
Can you add const reference functions too?

Reply:
`CTxdPool` has no const methods, const reference will be unused